apps/registry.json

```diff
@@ -51,23 +51,23 @@
         "id": "samourai-server",
         "category": "Wallet Servers",
         "name": "Samourai Server",
-        "version": "1.8.1",
+        "version": "1.9.0",
         "tagline": "Your private backing server for Samourai Wallet",
         "description": "Samourai Server is an exclusive Umbrel app that runs Samourai Dojo and Whirlpool backing servers, and provides you easy step-by-step instructions to connect your Samourai Wallet to them. Samourai Wallet is unrivaled in transaction privacy, but its default configuration is still subject to network level privacy loss.\n\nBy default, Samourai Wallet will connect to its centrally controlled Dojo hosted in Iceland. By hosting your own Samourai Server, you can bypass its servers completely when using Samourai Wallet or Sentinel, and gain total financial sovereignty without any complexity.",
         "developer": "Samourai",
         "website": "https://samouraiwallet.com",
         "dependencies": [
             "bitcoind"
         ],
-        "repo": "https://code.samourai.io/dojo/samourai-dojo",
+        "repo": "https://github.com/louneskmt/umbrel-samourai-dojo/tree/v1.9.0-umbrel",
         "support": "https://t.me/SamouraiWallet",
         "port": 3005,
         "gallery": [
             "1.jpg",
             "2.jpg",
             "3.jpg"
         ],
-        "path": "/connect/",
```

Comment:
Is this needed? `/connect/` works fine for me and I think it looks neater.

Reply:
`/connect/` doesn't work for me, I get a nginx 404.